caPutLogApp/caPutJsonLogTask.cpp

```diff
@@ -11,7 +11,11 @@
  * - Initial version.
 */
 
+// Windows needs this macro to avoid confict with min/max macros
+#define NOMINMAX
```

Comment:
While defining `NOMINMAX` shouldn't do anything on targets other than Windows, I've opted for a conditional definition.  eg. in [`configure/CONFIG_SITE`](https://github.com/mdavidsaver/pvxs/blob/2ab0d662bf3f74f32915d0e07383d8c30217fdd5/configure/CONFIG_SITE#L45).

```makefile
USR_CPPFLAGS_WIN32 += -DNOMINMAX
```

Reply:
+1

Reply:
Done.